rllib/agents/dqn/r2d2.py

```diff
@@ -56,14 +57,7 @@
         # and update that initial state during training (from the internal
         # state outputs of the immediately preceding sequence).
         "zero_init_states": True,
-        # If > 0, use the `burn_in` first steps of each replay-sampled sequence
-        # (starting either from all 0.0-values if `zero_init_state=True` or
-        # from the already stored values) to calculate an even more accurate
-        # initial states for the actual sequence (starting after this burn-in
-        # window). In the burn-in case, the actual length of the sequence
-        # used for loss calculation is `n - burn_in` time steps
-        # (n=LSTM’s/attention net’s max_seq_len).
-        "burn_in": 0,
+
```

Comment:
Let's set these to DEPRECATED_VALUE and add some validation checks, so users get a meaningful error message in case they are still using the old config keys.

Reply:
Absolutely, in the PR for ApeX I deprecate properly for all algorithms.
Sorry for this being a slight mess in the config dicts.